desktopexporter/config.go

```diff
@@ -9,8 +9,12 @@ type Config struct {
 	// Endpoint defines the host and port where we serve our frontend app
 	Endpoint string `mapstructure:"endpoint"`
 
-	// Endpoint defines the path of your database file. Setting an enpty string opens DuckDB in in-memory mode
+	// DbPath defines the path of your database file. Setting an enpty string opens DuckDB in in-memory mode
```

Comment:
```suggestion
	// DbPath defines the path of your database file. Setting an empty string opens DuckDB in in-memory mode
```

Reply:
I took these out of config so the problem solved itself. I opted to read them when we start the exporter, rather than the collector.